packages/components/slider/src/composables/use-slide.ts

```diff
@@ -5,8 +5,8 @@ import {
   UPDATE_MODEL_EVENT,
 } from '@element-plus/constants'
 import { useFormItem } from '@element-plus/components/form'
+import { type Arrayable, isArray } from '@element-plus/utils'
```

Comment:
`type` should be in a new line.

Reply:
I don't have this problem, it's strange.

Reply:
I know , the type should be outside {}.